src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/createaccessrequirement/CreateManagedACTAccessRequirementStep3.java

```diff
@@ -36,60 +20,16 @@ public class CreateManagedACTAccessRequirementStep3
   ModalPresenter modalPresenter;
   ManagedACTAccessRequirement accessRequirement;
   SynapseJavascriptClient jsClient;
-  AccessControlList originalAcl;
-  String reviewerPrincipalId;
-  UserTeamBadge reviewerUserTeamBadge;
-  public static final String TOO_MANY_ACL_ENTRIES_ERROR =
-    "This Access Requirment has an ACL that contains more than one item.  Beware, this dialog will destroy other entries on save.";
```

Comment:
The previous implementation allowed at most one additional reviewer to be added to the ACL. Per the new design, the SRC implementation doesn't restrict the number of users/teams with additional permissions